.github/workflows/docker.yaml

```
@@ -1,81 +1,78 @@
           platforms: linux/amd64,linux/arm64
+          target: bolt-ai-production
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          cache-from: type=registry,ref=${{ steps.string.outputs.lowercase }}:latest
-          cache-to: type=inline
-
-      - name: Check manifest
-        run: |
-          docker buildx imagetools inspect ${{ steps.string.outputs.lowercase }}:${{ steps.meta.outputs.version }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)